src/comment.ts

```diff
@@ -170,8 +174,8 @@ export const comment = async () => {
 
     const [noteInfo, hasChangeset, { changedPackages, releasePlan }] =
       await Promise.all([
-        getNoteInfo(api, mrIid),
-        hasChangesetBeenAdded(changedFilesPromise),
+        await getNoteInfo(api, mrIid),
```

Comment:
These `await` seem unnecessary

Reply:
You are absolutely right. I've changed it.